File: fstmerge/examples/Stack_python/Size/stack.py
class Stack:
    items = list()
    
    def push(self, item):
        self.items.append(item)
    
    def size(self):
        return len(self.items)
    
    def pop(self):
        if len(self.items) > 0:
            return self.items.pop()
        else:
            return None

File: fstmerge/examples/Stack/Base/Stack.java
import java.util.LinkedList;

public class Stack<T> {
	private LinkedList<T> items = new LinkedList<T>();
	public void push(T item) {
		items.addFirst(item);
	}
	public T pop() {
		if(items.size() > 0)
			return items.removeFirst();
		else
			return null;
	}
}